api/migrations/20211012181841-createContributions.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        return queryInterface.sequelize.transaction(t => {
+            return Promise.all([
+                queryInterface.addColumn('Contributions', 'contributor_id', {
```

Comment:
If you try adding a column to a table that does not exist, it will throw you an error.
You need to create de table with the createTable and inside o that function add the new columns.
https://sequelize.org/master/manual/migrations.html
```suggestion
                queryInterface.addColumn('Contributions', 'contributor_id', {
```